test/integration/price-import.spec.coffee

```diff
@@ -87,16 +87,24 @@ describe 'Price Importer integration tests', ->
     prices = [
       {
         sku: 'sku1'
-        value:
-          centAmount: 9999
-          currencyCode: 'EUR'
+        prices: [
+          {
+            value:
+              centAmount: 9999
+              currencyCode: 'EUR'
+          }
+        ]
```

Comment:
So we are now aggregating prices for each variant (sku) ? I thought we wanted to do it for the all product...